apps/wallet/src/ui/app/pages/home/transfer-coin/SendTokenForm.tsx

```diff
@@ -52,54 +45,47 @@ export type SendTokenFormProps = {
     initialGasEstimation: number;
 };
 
-function GasBudgetEstimation({
-    coinDecimals,
-    suiCoins,
-}: {
-    coinDecimals: number;
-    suiCoins: CoinStruct[];
-}) {
-    const { values, setFieldValue } = useFormikContext<FormValues>();
-    const gasBudgetEstimationUnits = useMemo(
-        () =>
-            Coin.computeGasBudgetForPay(
-                suiCoins,
-                parseAmount(values.amount, coinDecimals)
-            ),
-        [coinDecimals, suiCoins, values.amount]
-    );
-    const { gasBudget: gasBudgetEstimation, isLoading } = useGasBudgetInMist(
-        gasBudgetEstimationUnits
-    );
+// TODO: Rebuild gas budget estimation:
+// function GasBudgetEstimation({
+//     coinDecimals,
+//     suiCoins,
+// }: {
+//     coinDecimals: number;
+//     suiCoins: CoinStruct[];
+// }) {
+//     const { values, setFieldValue } = useFormikContext<FormValues>();
+//     const { gasBudget: gasBudgetEstimation, isLoading } = useGasBudgetInMist(
+//         gasBudgetEstimationUnits
+//     );
 
-    const [formattedGas, gasSymbol] = useFormatCoin(
-        gasBudgetEstimation,
-        GAS_TYPE_ARG
-    );
+//     const [formattedGas, gasSymbol] = useFormatCoin(
+//         gasBudgetEstimation,
+//         GAS_TYPE_ARG
+//     );
 
-    // gasBudgetEstimation should change when the amount above changes
-    useEffect(() => {
-        setFieldValue('gasInputBudgetEst', gasBudgetEstimation, true);
-    }, [gasBudgetEstimation, setFieldValue, values.amount]);
+//     // gasBudgetEstimation should change when the amount above changes
+//     useEffect(() => {
+//         setFieldValue('gasInputBudgetEst', gasBudgetEstimation, true);
```

Comment:
It seems that now that we don't update the budget the validation fails. Removing `!!gasBudgetInput &&` from [here](https://github.com/MystenLabs/sui/blob/a343ec697eedce9315a88f2faeb5e6e126b466e3/apps/wallet/src/shared/validation.ts#L74) should fix it. Not sure if we need this check now so we could remove all of it

Reply:
Updated this.